build.gradle.kts

```diff
@@ -74,6 +74,7 @@ subprojects {
         languageVersion = "1.4"
         jvmTarget = "11"
         freeCompilerArgs += "-Xjvm-default=enable"
+        freeCompilerArgs += "-Xopt-in=kotlin.ExperimentalStdlibApi"
```

Comment:
Required to access coroutine context elements, which surprisingly is an experimental API. 🤷‍♂️